pkg/asset/agent/image/agentartifacts.go

```diff
@@ -64,19 +65,31 @@ func (a *AgentArtifacts) Generate(dependencies asset.Parents) error {
 	a.RendezvousIP = ignition.RendezvousIP
 	a.IgnitionByte = ignitionByte
 	a.ISOPath = baseIso.File.Filename
+
+	platform := agentManifests.AgentClusterInstall.Spec.PlatformType
+	var platformName string
+	if agentManifests.AgentClusterInstall.Spec.ExternalPlatformSpec != nil {
+		platformName = agentManifests.AgentClusterInstall.Spec.ExternalPlatformSpec.PlatformName
+	}
+	var skipNetworkTUI bool
+	if platform == hiveext.ExternalPlatformType && platformName == "oci" {
```

Comment:
Maybe we could add an `ExternalPlatformName()` function or something to AgentClusterInstall, since this code has to be repeated here and for the kargs.

Reply:
Addded via https://github.com/openshift/installer/pull/7511/files#diff-ed3ddff72757f2b9f8078d820e70ac801ad04c31885e033d6722ac6ae865f10e